commands-yml/commands/device/app/install-app.yml

```diff
@@ -40,7 +40,7 @@ client_docs:
   ruby: "https://www.rubydoc.info/github/appium/ruby_lib_core/Appium/Core/Device#install_app-instance_method"
   ruby_core: "https://www.rubydoc.info/github/appium/ruby_lib_core/Appium/Core/Device#install_app-instance_method"
   php: "https://github.com/appium/php-client/" # TODO PHP documentation link
-  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO Dotnet documentation link
+  csharp: "https://github.com/appium/appium-dotnet-driver/blob/master/src/Appium.Net/Appium/AppiumDriver.cs#L227"
```

Comment:
Please do not use line numbers in links. These might easily get outdated if the file is changed

Reply:
Good point, I've done this throughout so will remove them 👍

Reply:
Should be sorted now, thanks for the feedback 😄